x-pack/plugins/security_solution/common/endpoint/service/authz/authz.ts

```diff
@@ -223,6 +225,7 @@ export const calculateEndpointAuthz = (
     canReadPolicyManagement,
     canWriteActionsLogManagement,
     canReadActionsLogManagement: canReadActionsLogManagement && isEnterpriseLicense,
+    canAccessEndpointActionsLogManagement: canReadActionsLogManagement && isPlatinumPlusLicense,
```

Comment:
Maybe we could name this `canAccessEndpointListActionsLog`? @paul-tavares.

Reply:
I'm ok introducing the additional property, but can we name it consistency with how other are named? So in this case, it would be something like `canReadSingleEndpointAcitonLog` (notice I also renamed it a bit).

Also, looking at this change, are we saying that Action logs can be seen in Endpoint Details panel if license is Platinum++, but Action Log page is Enterprise?

Reply:
Actually - never mind. I think the naming you created is good. I see there is precedence. Sorry for the noise.